### ISSUE TITLE
Unable to scroll if user clicks deny on the enter full screen info bar

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#3837]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Unable to scroll if user clicks deny on the enter full screen info bar

**Expected behavior:**
User should be able to scroll up/down on the page irrespective of whether allow/deny selection for full screen
- Platform (Win7, 8, 10? macOS? Linux distro?):
  Windows 10 x64
- Brave Version:
  0.12.0
- Steps to reproduce:
  1. Play video on Youtube/Vimeo
  2. Click on Full screen button
  3. Click Deny on the info bar at the top
  4. Unable to scroll after selecting deny. Need to navigate to  different page or refresh page to scroll down.
  5. Unable to enter full screen again. Have to refresh the page to enter full screen.
- Screenshot if needed:
- Any related issues:

**Comment by eljuno** [comment]
Duplicate of #3424?

**Comment by srirambv** [comment]
Yup. Closing the issue as dupe of #3424
